[CI] EsqlNodeFailureIT testFailureLoadingFields failing

**Build Scans:**
- [elasticsearch-periodic #6192 / encryption-at-rest](https://gradle-enterprise.elastic.co/s/yrupwymecbikk)
- [elasticsearch-periodic-platform-support #6181 / sles-12_platform-support-unix](https://gradle-enterprise.elastic.co/s/zbyp2m2x3zoi2)
- [elasticsearch-periodic-platform-support #6160 / oraclelinux-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/tefypfvmzt6ss)
- [elasticsearch-periodic-platform-support #6160 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/wpcikblzaqamk)
- [elasticsearch-periodic #6157 / single-processor-node-tests](https://gradle-enterprise.elastic.co/s/mt2ofmwjuf2uw)
- [elasticsearch-periodic-platform-support #6146 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/2l5f5kogwov2i)
- [elasticsearch-periodic-platform-support #6125 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/sfbja64gfedtq)
- [elasticsearch-periodic-platform-support #6118 / debian-11_platform-support-unix](https://gradle-enterprise.elastic.co/s/a3updgraogufg)
- [elasticsearch-periodic-platform-support #6111 / oraclelinux-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/hjmeqw4nd2p5m)
- [elasticsearch-periodic-platform-support #6097 / rocky-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/3t6h35vjd55q4)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.action.EsqlNodeFailureIT.testFailureLoadingFields" -Dtests.seed=F438992932E3218E -Dtests.locale=fo-Latn-FO -Dtests.timezone=America/Eirunepe -Druntime.java=23
```

**Applicable branches:**
8.17

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.action.EsqlNodeFailureIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testFailureLoadingFields),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Leftover exchanges ExchangeService{sinks=[07-RWrYjTJaNx9cHqrPmig:804/2]} on node node_s0
```

**Issue Reasons:**
- [8.17] 2 consecutive failures in test testFailureLoadingFields
- [8.17] 12 failures in test testFailureLoadingFields (3.5% fail rate in 345 executions)
- [8.17] 2 failures in step oraclelinux-8_platform-support-unix (22.2% fail rate in 9 executions)
- [8.17] 3 failures in step amazonlinux-2_platform-support-aws (25.0% fail rate in 12 executions)
- [8.17] 2 failures in pipeline elasticsearch-periodic (15.4% fail rate in 13 executions)
- [8.17] 7 failures in pipeline elasticsearch-periodic-platform-support (53.8% fail rate in 13 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -147,9 +147,6 @@ tests:
   issue: https://github.com/elastic/elasticsearch/issues/118914
 - class: org.elasticsearch.xpack.security.authc.ldap.ActiveDirectoryRunAsIT
   issue: https://github.com/elastic/elasticsearch/issues/115727
-- class: org.elasticsearch.xpack.esql.action.EsqlNodeFailureIT
-  method: testFailureLoadingFields
-  issue: https://github.com/elastic/elasticsearch/issues/118000
 - class: org.elasticsearch.index.mapper.AbstractShapeGeometryFieldMapperTests
   method: testCartesianBoundsBlockLoader
   issue: https://github.com/elastic/elasticsearch/issues/119201

